vendor/bat-native-ledger/src/bat/ledger/internal/database/database_pending_contribution.cc

```diff
@@ -67,7 +67,9 @@ void DatabasePendingContribution::InsertOrUpdateList(
       _1,
       callback);
 
-  ledger_->RunDBTransaction(std::move(transaction), transaction_callback);
+  ledger_->GetLedgerClient()->RunDBTransaction(
```

Comment:
Does it make more sense to pass a `LedgerClient` to some of these classes now instead of a `LedgerImpl`?

Reply:
I would still go with ledger_impl as we have more moves like this so ledger_impl will hold all objects that you can use

Reply:
OK, just wanted to check, sounds good.